### ISSUE TITLE
Inference of Multiband MelGAN (v2) with ForwardTacotron

### ISSUE THREAD

**Issue by prajwaljpj** [kan-bayashi/ParallelWaveGAN#346]
I have trained a [ForwardTacotron](https://github.com/as-ideas/ForwardTacotron) text2mel model and I would like to integrate it into Parallelwavegan. 
For now we have extracted the genrated mel_post (Melspectrogram after Postnet) from [here](https://github.com/as-ideas/ForwardTacotron/blob/3bcaf3569ea2379ff995403b31f280720df3f03d/models/forward_tacotron.py#L180) and saving it as a .npy file ([alifiya_esp_1.npy.zip](https://github.com/kan-bayashi/ParallelWaveGAN/files/8315314/alifiya_esp_1.npy.zip)). Then we use StandardScalar to normalize the data from here ([1](https://github.com/kan-bayashi/ParallelWaveGAN/blob/e027f53ee7c5dc813d61cf3a47749a6e2abc9369/parallel_wavegan/bin/normalize.py#L165-L175) and [2](https://github.com/kan-bayashi/ParallelWaveGAN/blob/e027f53ee7c5dc813d61cf3a47749a6e2abc9369/parallel_wavegan/bin/normalize.py#L185)) and infer through [here](https://github.com/kan-bayashi/ParallelWaveGAN/blob/e027f53ee7c5dc813d61cf3a47749a6e2abc9369/parallel_wavegan/bin/decode.py#L147-L173). 
This is the sample output of the corresponding numpy file ([alifiya_esp_1.wav.zip](https://github.com/kan-bayashi/ParallelWaveGAN/files/8315368/alifiya_esp_1.wav.zip))
The same mel is working fine with GriffinLim.

Where am I going wrong?

**Comment by kan-bayashi** [comment]
Not sure but the following points might be different:

- log basis (I use log10 as a default)
- fmin and fmax for mel basis (I use 80-7600 as a default)
- normalization (I use mean var normalization using stats of training data)

If you use the same mel basis, maybe log is different.
https://github.com/as-ideas/ForwardTacotron/blob/3bcaf3569ea2379ff995403b31f280720df3f03d/utils/dsp.py#L71-L87

You can change the log basis to match the feature extraction condition.

**Comment by prajwaljpj** [comment]
Is there a way to circumvent without re-training the text2mel model?

**Comment by redhood95** [comment]
> * log basis (I use log10 as a default)
> * normalization (I use mean var normalization using stats of training data)

we have  tried to fix this by using np.log10(np.exp(ft_mel_output)) and normalize it using the mean var normalization  with standardScaler
are there any other changes we can make at the inference time for fixing this ?

**Comment by kan-bayashi** [comment]
If fmin / fmax are different, there is no way to use pretrained model.
